configure.ac

```diff
@@ -226,5 +235,6 @@ echo -e "Build with font-engine support ..... : $font_engine_name"
 echo -e "Build with fontconfig support ...... : $want_fontconfig"
 echo -e "Build with thread support .......... : $thread_name"
 echo -e "Build with video support ........... : $video_driver_name"
+echo -e "Build with optimized scaling ....... : $enable_bilinear_scaling"
```

Comment:
No changes are needed here, you just need to add Graph_Zoom_Bilinear(), I will make some changes in the future to get it called at the right time.